boefjes/boefjes/plugins/kat_webpage_analysis/main.py

```diff
@@ -66,13 +67,36 @@ def run(boefje_meta: BoefjeMeta) -> list[tuple[set, bytes | str]]:
         if content_type[0] in ALLOWED_CONTENT_TYPES:
             body_mimetypes.add(content_type[0])
 
+    # in case of a full response object, we hexdump to avoid issues with binary data or different encoding
+    response_dump = json.dumps(create_response_object(response))
+    content = binascii.hexlify(response.content).decode()
+
     return [
-        ({"openkat-http/full"}, f"{response.headers}\n\n{response.content}"),
-        ({"openkat-http/headers"}, json.dumps(dict(response.headers))),
+        ({"openkat-http/response"}, f"{response_dump}\n\n{content}"),
+        ({"openkat-http/headers"}, response_dump),
```

Comment:
This changes the format of openkat-http/headers raw files which will result in that the changed headers normalizer won't be able to parse the old raw files. Also if other people have created their own normalizers this will also break. I don't think we should just change the format of raw files this way.